patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -5534,6 +5648,15 @@
  			if ((!overrideCanPlace.HasValue || overrideCanPlace.Value) && TileObjectData.CustomPlace(tileToCreate, previewPlaceStyle))
  				TileObject.CanPlace(tileTargetX, tileTargetY, tileToCreate, previewPlaceStyle, direction, out var _, onlyCheck: true, forcedRandom);
  		}
+@@ -33782,7 +_,7 @@
+ 		overrideCanPlace = null;
+ 		forcedRandom = null;
+ 		if (UsingBiomeTorches && tileToCreate == 215 && previewPlaceStyle == 0)
```

Comment:
is this meant to have a `TileID.Sets.Campfires` check instead of `215`?

Reply:
It is not, this is specifically checking for the original campfire item, since that is the only one that will be changed by the biome campfire feature.